packages/@okta/vuepress-site/guides/sign-into-your-web-app/configure-packages/aspnet/addconfigpkg.md

```diff
@@ -0,0 +1,20 @@
+All of our .NET SDKs are hosted on [NuGet](https://www.nuget.org/). These instructions show you how to install all of the dependencies that you need to configure your web application to use the Okta middleware.
+
+Install the following dependencies in your project:
+
+* [Microsoft.Owin.Security.Cookies](https://www.nuget.org/packages/Microsoft.Owin.Security.Cookies) 4.0.0 or higher 
+* [Microsoft.Owin.Host.SystemWeb](https://www.nuget.org/packages/Microsoft.Owin.Host.SystemWeb) 4.0.0 or higher
+* [Okta.AspNet](https://www.nuget.org/packages/Okta.AspNet)
+
+You can install them via the **NuGet Package Manager**:
+
+1. In the Solution Explorer, right-click on your solution and select **Manage Nuget Packages for Solution**.
+1. Click **Browse** and search for the package that you want to install, for example, `Okta.AspNet`.
```

Comment:
the numbering here needs to be fixed.

Reply:
> the numbering here needs to be fixed.

For ordered lists, you can just declare (1.) and VuePress will do the magic for you :)
https://vuepressbook.com/tutorial/tutorial4.html#markdown-lists

Reply:
d'oh! Thank you! I just built it locally and all is good. Should've done that first. My bad. Thanks!